packages/ui/src/theme/css/component/sliderField.scss

```diff
@@ -91,18 +109,17 @@
   background-color: var(
     --amplify-components-sliderfield-range-background-color
   );
-  
+
   &[data-disabled] {
```

Comment:
Why not flatten `disabled`?

Reply:
I think it's because we can't access the underlying Radix component to add our own flattened classes.